synced_versions_formulae.json

```diff
@@ -13,6 +13,7 @@
   ["avro-c", "avro-cpp", "avro-tools"],
   ["aws-console", "cfn-format", "rain"],
   ["b3sum", "blake3"],
+  ["bcftools", "htslib", "samtools"],
```

Comment:
These will need to be removed from `autobump.txt` since `brew bump` doesn't support synced versions formulae.